### PR TITLE
Removed *nix local exec tasks from Terraform config

### PR DESCRIPTION
**Pull request by Duffney**
Improve the usability of the Terraform configuration by removing tasks that depend on a *nix system to run them. Tasks that were taken out of the Terraform configuration were put as manual tasks within the readme file.

### CODE REVIEW

**Comment by shizhMSFT** [comment]
> @shizhMSFT Do you know who can help us to review this PR?

@dtzar Could you provide context as you have reviewed related PRs?

File: docs/nv2-terraform.md [hunk 1]
```diff
@@ -60,15 +60,86 @@ Run the following commands:
     azure_resource_group_name = "notary-4d32ffd6-rg"
     signing_key_name = "example"
     ```
+6. Configure Kubectl
```
Comment (dtzar) [inline docs/nv2-terraform.md:63]: I would recommend adding in a step between 4 and 5 which has people populate a (please create a new one in the PR) [TFVARS file](https://developer.hashicorp.com/terraform/language/values/variables#variable-definitions-tfvars-files).

**Comment by dtzar** [comment]
Other than my one suggestion, I think the code itself looks good.  What I don't have time for is to test this out.  I would just have someone test this on a mac/ubuntu and then a Windows machine and make sure the steps work.  If so, then good to merge.

**Comment by shizhMSFT** [comment]
Closing this PR since `iac` has been deleted by #99

**Pull request closed without merging**